src/xcode/ENA/ENA/Source/Extensions/ENTemporaryExposureKey+Processing.swift

```diff
@@ -41,21 +41,24 @@ extension Array where Element: ENTemporaryExposureKey {
 			return daysUntilToday
 		})
 
-		/// 2. Assign the corresponding transmission risk levels and filter out keys that have no corresponding transmission risk level
-		for (daysUntilToday, exposureKeys) in groupedExposureKeys {
-			if daysUntilToday >= 0 && daysUntilToday <= 14 {
-				for exposureKey in exposureKeys {
-					/// Assign corresponding transmission risk level
-					exposureKey.transmissionRiskLevel = symptomsOnset.transmissionRiskVector[daysUntilToday]
-				}
-			} else {
-				/// Remove keys that have no corresponding transmission risk level
-				groupedExposureKeys[daysUntilToday] = nil
+		/// 2. Assign the corresponding transmission risk levels and days since onset of symptoms. Keys that have no corresponding transmission risk level are filtered out.
+		var processedExposureKeys = [SAP_TemporaryExposureKey]()
+		for (daysUntilToday, exposureKeys) in groupedExposureKeys where daysUntilToday >= 0 && daysUntilToday <= 14 {
```

Comment:
It might be worth considering renaming `daysUntilToday` to `ageInDays`. For me, it makes it a bit clearer that I cannot really be negative.

Reply:
I like that name suggestion, changed it 👍